File: Day 23/Explicit_Intent1/app/src/main/java/com/example/explicit_intent_1/MainActivity.java
package com.example.explicit_intent_1;

import androidx.appcompat.app.AppCompatActivity;



import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button click;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        click = findViewById(R.id.click);

        Intent intent = new Intent(MainActivity.this,Second.class);
        intent.putExtra("title","home");
        intent.putExtra("name","mayur");
        intent.putExtra("rollno",10);

        click.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(intent);
            }
        });


    }
}

File: Siddhi mam Tasks/Day 20/Layouts/app/src/main/java/com/example/layouts/MainActivity.kt
package com.example.layouts

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button



class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val layout1 = findViewById<Button>(R.id.layout1)

        layout1.setOnClickListener {
            // Switching from ActivityA to SecondActivity
            val intent = Intent(this,Screen_1::class.java)
            startActivity(intent)
        }





    }
}